### ISSUE TITLE
DISCUSSIONS: When not logged in, argument 'reply' buttons are missing

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1637]
@ssin1901 @ilmartyrk A similar issue to [this one](https://github.com/citizenos/citizenos-fe/issues/1636), maybe both could be fixed together. 

**What is the problem?**
Currently, when you are not logged in, the reply buttons on arguments in public topics are not visible: 
![image](https://github.com/user-attachments/assets/4ad9f13c-2d68-45e6-8155-6a10beca245e)

**Why is this a problem?**
This will be very confusing for users, especially first-time users, who will (IMO) not know that a button is missing, and either not know they should be able to reply (if there are no replies), or not understand why others are able to reply and not them. 

It's also inconsistent, as the 'Add an argument' button is visible when logged out.    

**Possible solution.**
It should work the same as the 'Add an argument' button, and look/work the same as it does when logged in (blue-black with hover state), but when you click on it the 'Login' pop-up appears. 

Will add this one to 'Next' along with #1636 as seems quite important - could be causing issues for public topics that are currently active.

**Comment by kevincrepin** [comment]
Yes, I agree. The buttons should remain there as CTAs and trigger the login popup.

**Comment by BeccaMelhuish** [comment]
Adding 'grouped' label and moving to 'no status' column as it is now covered in https://github.com/citizenos/citizenos-fe/issues/1706

**Comment by BeccaMelhuish** [comment]
**Dev task summary**

- [ ] Make sure reply button still shows on arguments in public topics even when logged out
- [ ] Make clicking the button activate the log in pop-up (with new text as per [1640](https://github.com/citizenos/citizenos-fe/issues/1640))

**Comment by BeccaMelhuish** [comment]
@ssin1901 checked on desktop and am happy with it :)

**Comment by ssin1901** [comment]
@DiverVM @BeccaMelhuish I checked and looks ok. Did a quick check in mobile as well and works fine.

Clicking reply as a logged out user prompted login popup and login went fine.

https://github.com/user-attachments/assets/de61ea33-502b-44bd-a4a5-463f7aad5a62

**Comment by ssin1901** [comment]
Moving to Pre-Live
